server/routes/cards.js

```diff
@@ -0,0 +1,66 @@
+const express = require('express');
+const router = express.Router();
+const Card = require('../../database/models/Card');
+
+router.post('/', (req, res) => {
+  const newCard = {
+    title: req.body.title,
+    priority_id: parseInt(req.body.priority),
+    status_id: 1
+  };
+
+  new Card(newCard)
+    .save()
+    .then(() => {
+      return res.send('task posted succesfuly');
+    })
+    .catch(err => {
+      return res.send(err);
+    });
+});
+
+router.get('/', (req, res) => {
+  Card.fetchAll()
+    .then(cards => {
+      res.json(cards);
+    })
+    .catch(err => {
+      res.send(err);
+    });
+});
+
+router.put('/', (req, res) => {
```

Comment:
So typically a put takes in an id in the route, so you're route should look like 

`'/:id`

And basically you use that in your `Card.where`, putting the id in the body is optional at that point.